### ISSUE TITLE
About Draft functional needs linked from WCAG 3.0 

### ISSUE THREAD

**Issue by spanchang** [w3c/silver#221]
Reference: https://www.w3.org/WAI/GL/WCAG3/2020/functional-needs/

Is it possible to include an introductory paragraph that says why this level of detail  is needed? 
Does when one became disabled or how or level of permenance impact the outcome statement  or remediation method? 

Refer to items like:
Use without vision from birth then without hearing as acquired 
Use without hearing from birth then without vision as acquired 
Use as blind (born without vision)
Use with blindness (acquired blindness during lifetime)
Use with limited central vision
Use with limited peripheral vision ·XR·
Use with limited interocular acuity or monocular input

Does "use with limited vision" not cover limited field of vision, inability to perceive depth, besides low vision?   Perhaps these phrases can be used to describe "limited vision"

I draw attention to the note in Clause 4 of EN 301 549: 
"NOTE 3: The methods of meeting the accessibility needs of users with multiple impairments will depend on the specific combination of impairments. Meeting
these user accessibility needs may be addressed by considering multiple clauses in clause 4.2." 
Also, it is well established that a solution that helps one user group  may often help other user groups  of PWD.   
Surely it is justified to include a condition like "not able to see color" as a distinct need as it can impact some individuals without  impacting their vision generally. 
So if there is a category like color vision deficit that has not been  considered with regard to the different disabilities, surely that should be listed.

We need to be mindful that the level of detail does not impair the usability of the doc.
Thanks and humbly,
Sailesh

**Comment by jspellman** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by fstrr** [comment]
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  The Accessibility Guidelines Working Group attempted to address these concerns in this latest draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by spanchang** [comment]
Thanks very much for your consideration. I will do as suggested.
Best,
Sailesh
